lib/api/v1/settings-controller.js

```diff
@@ -35,6 +35,14 @@ app.put('/api/v1/settings/fetching/:op', user.can('change settings'), function(r
 
 });
 
+// Get Google Places API Key. Remember to set it so that only certain domains can use it
```

Comment:
@andresfib Copying your response from the other thread:

"Everyone accessing the Aggie deployment will be able to see the key in the source code of index.html. People should whitelist the addresses that can use the key so 'unathorized' requests are not taken into account by Google. By signing up we get 1K free requests per day, by adding a credit card we get 150K.
https://developers.google.com/places/web-service/usage

This may go in the README..."